warehouse/upload.go

```diff
@@ -232,6 +241,31 @@ func (job *UploadJobT) generateUploadSchema(schemaHandle *SchemaHandleT) error {
 	return err
 }
 
+// excess columns are included in excluded schema
+func (job *UploadJobT) generateExcludedSchema() {
+	if !enableExcludedSchema {
+		return
+	}
+	uploadSchema := job.upload.UploadSchema
+	if job.upload.LoadFileType == warehouseutils.LOAD_FILE_TYPE_PARQUET {
+		uploadSchema = job.upload.MergedSchema
+	}
+
+	var excludedSchema warehouseutils.SchemaT
+	if maxColumnCount, ok := maxColumnCounts[job.upload.DestinationType]; ok {
+		excludedSchema = GetExcludedSchema(uploadSchema, job.schemaHandle.localSchema, maxColumnCount)
+	}
+
+	if len(excludedSchema) > 0 {
+		pkgLogger.Infof("Exclude schema for upload id %d: %v", job.upload.ID, excludedSchema)
```

Comment:
This might generate a lot of noise in the logs. Should we log the count only or is there a need to log the type and column name as well ?

Reply:
This log gets printed only when upload exceeds maximum column limit. Added for easy visibility of schema.